adoc/chapters/device_compiler.adoc

```diff
@@ -737,8 +737,8 @@ The variable declarations get assigned to an address space depending on their
 scope and storage class:
 
   * Namespace scope
-  ** If the type is [code]#const#, the address space the declaration is assigned
-     to is implementation-defined.
+  ** If the type is [code]#const#, the address space of the declaration is
+     assigned to is implementation-defined.
```

Comment:
This wording no longer makes sense.  The original wording was a little awkward, but it did make sense.  Are we trying to say this:

> If the type is `const`, the declaration is assigned to an implementation-defined address space.

Reply:
reads better, I'll apply this.